DESCRIPTION

```diff
@@ -28,6 +28,7 @@ Imports:
     lifecycle,
     openssl (>= 2.0.0),
     packrat (>= 0.6),
+    prettyunits,
```

Comment:
I only use this in one place, but it's also a very lightweight dep. I filed a cli issue so we can eventually get rid of it: https://github.com/r-lib/cli/issues/588